docs/introduction/installation.mdx

```diff
@@ -398,6 +398,31 @@ All available values are listed on the [artifacthub](https://artifacthub.io/pack
 
 ---
 
+## Installation as a Home Assistant Add-on
+
+Homarr can now be seamlessly integrated into your Home Assistant setup with this add-on. This is perfect for users who rely on Home Assistant as their primary server and don't have a separate device for hosting.
+
+### Prerequisites
+
+- [Home Assistant](https://www.home-assistant.io/getting-started/)
```

Comment:
Not all instances support addons.. Please mention this here

Reply:
Updated